packages/radix-ui-themes/src/styles/utilities/padding.css

```diff
@@ -1,331 +1,505 @@
 @breakpoints {
+  .rt-r-p {
+    /* Keep this selector here to maintain the source order */
+  }
   .rt-r-p-0 {
     padding: 0;
   }
-
   .rt-r-p-1 {
     padding: var(--space-1);
   }
-
   .rt-r-p-2 {
     padding: var(--space-2);
   }
-
   .rt-r-p-3 {
     padding: var(--space-3);
   }
-
   .rt-r-p-4 {
     padding: var(--space-4);
   }
-
   .rt-r-p-5 {
     padding: var(--space-5);
   }
-
   .rt-r-p-6 {
     padding: var(--space-6);
   }
-
   .rt-r-p-7 {
     padding: var(--space-7);
   }
-
   .rt-r-p-8 {
     padding: var(--space-8);
   }
-
   .rt-r-p-9 {
     padding: var(--space-9);
   }
-
-  .rt-r-p-current {
+  .rt-r-p-inset {
     padding: var(--inset-padding);
   }
-
+  .rt-r-px {
+    /* Keep this selector here to maintain the source order */
```

Comment:
OK as I started to reply to this comment and wanted to copy-paste an example of the resulting CSS after the transformations, I realized that one had a significant flaw and reworked some stuff to fix it. Good question 🙂 

Anyway, in the resulting CSS, we want to have all related `.xs:`, `.md:`, `.lg:`, etc rules grouped together like this:

```css
.rt-r-inset { ... }
.rt-r-inset-0 { ... }
.rt-r-inset-1 { ... }
...

@media (--xs) {
  .xs:rt-r-inset { ... }
  .xs:rt-r-inset-0 { ... }
  .xs:rt-r-inset-1 { ... }
  ...
}

@media (--md) {
  .md:rt-r-inset { ... }
  .md:rt-r-inset-0 { ... }
  .md:rt-r-inset-1 { ... }
  ...
}

...
```

Adding an empty rule inside @breakpoints allows us to take advantage of how our PostCSS setup combines the duplicate selectors, so I'm deliberately placing the matching selectors in the right spot while they are defined later below @breakpoints.

Otherwise, without placing the rule in the right spot first, the result would have been this:

```css
.rt-r-inset-0 { ... }
.rt-r-inset-1 { ... }
...

@media (--xs) {
  .xs:rt-r-inset-0 { ... }
  .xs:rt-r-inset-1 { ... }
  ...
}

@media (--md) {
  .md:rt-r-inset-0 { ... }
  .md:rt-r-inset-1 { ... }
  ...
}
...

.rt-r-inset { ... }

@media (--xs) {
  .xs:rt-r-inset { ... }
}

@media (--md) {
  .md:rt-r-inset { ... }
}
...
```

This would have made it so that rules like `rt-r-inset` or `xs:rt-r-inset` would override the preceding hardcoded values like `lg:rt-r-inset-0`